include/common/postgres_compat.h

```diff
@@ -15,6 +16,7 @@
 
 #define lengthof(array) (sizeof (array) / sizeof ((array)[0]))
 #define pg_hton32(x) htobe32(x)
+#define pg_strncasecmp strncasecmp
```

Comment:
These aren't necessarily the same thing. `pg_strncasecmp()` only case-folds ASCII characters. Maybe that's ok for current use? But we should be careful about documenting this.